src/stringBuffer.js

```diff
@@ -29,7 +29,15 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let str = '';
+
+  function buffer(enotherString = '') {
```

Comment:
firstly it's `Another`
secondly it's better to rename it to `value` for example

Reply:
I urgently have to attend an English lesson